library/general/src/lib/installation/proposal_client.rb

```diff
@@ -116,6 +116,11 @@ def run
     #   submodules that translate any strings internally based on internal maps
     #   (e.g., keyboard etc.) need to take more action.
     #
+    # @option attrs [Boolean] "cramped_label_mode"
```

Comment:
What about something like `simple_mode`? `cramped_label_mode` sounds a bit complicated to me...